Header header
string node_id
bool for_console
int8 priority
string status_text
